### ISSUE TITLE
IL2CPU appears to have missing or invalid Plug Methods.

### ISSUE THREAD

**Issue by OtterConsultant** [CosmosOS/Cosmos#3054]
Have you checked Github Issues for similar errors?
Yes, there appears to be missing plugs in IL2CPU

**Exception**
Post the exception returned by Visual Studio
`Warning (active) Invalid plug method! Target method not found. : A0SystemObjectA2CosmosCore_PlugsSystemActivatorImplCreateInstanceA2CosmosCoreCosmosRuntimeTypeA0SystemBooleanA0SystemBoolean	Replica	C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1`

`Warning (active) Invalid plug method! Target method not found. : A0SystemInt64A3CosmosSystem_PlugsSystemIOOSFileStreamStrategyImplSeekA0SystemInt64A0SystemObject
 Replica C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1`

`Warning (active) Invalid plug method! Target method not found. : A0SystemVoidA3CosmosSystem_PlugsSystemIOStreamWriterImplWriteFormatHelperA0SystemStringA0A0SystemObjectarrayA0SystemBoolean	Replica	C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1`

**Visual Studio Output Logs**
```
Warning (active) Loading plugs from assembly: Cosmos.Core_Asm, Version=10.0.0.0, Culture=neutral, PublicKeyToken=5ae71220097cb983	Replica	C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1
Warning (active) Loading plugs from assembly: Cosmos.Core_Plugs, Version=10.0.0.0, Culture=neutral, PublicKeyToken=5ae71220097cb983	Replica	C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1
Warning (active) Loading plugs from assembly: Cosmos.Debug.Kernel.Plugs.Asm, Version=10.0.0.0, Culture=neutral, PublicKeyToken=5ae71220097cb983	Replica	C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1
Warning (active) Loading plugs from assembly: Cosmos.System2_Plugs, Version=10.0.0.0, Culture=neutral, PublicKeyToken=5ae71220097cb983	Replica	C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1	
Warning (active) Invalid plug method! Target method not found. : A0SystemObjectA2CosmosCore_PlugsSystemActivatorImplCreateInstanceA2CosmosCoreCosmosRuntimeTypeA0SystemBooleanA0SystemBoolean	Replica	C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1
Warning (active) Invalid plug method! Target method not found. : A0SystemInt64A3CosmosSystem_PlugsSystemIOOSFileStreamStrategyImplSeekA0SystemInt64A0SystemObject
 Replica C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1
Warning (active) Invalid plug method! Target method not found. : A0SystemVoidA3CosmosSystem_PlugsSystemIOStreamWriterImplWriteFormatHelperA0SystemStringA0A0SystemObjectarrayA0SystemBoolean	Replica	C:\Users\aajfr\source\repos\Replica\Replica\IL2CPU	1
```

**How To Reproduce**
Compile a new Cosmos Kernel

**Screenshots**
![image](https://github.com/user-attachments/assets/7d253111-cff7-415f-b74d-25b9b0650867)

**Context**
Before posting please confirm that the following are in order
[Y] Both Cosmos VS Extensions are installed
[Y] In the NuGet Package Manager "Include prerelease" is selected
[Y] The Cosmos NuGet package store is selected (NOT nuget.org) in 'Manage NuGet Packages'
[Y] The Cosmos NuGet packages are installed

Add any other context about the problem which might be helpful.

The kernel does compile and seems to run fine in my tests, however, i am unsure if these will be a problem later on as my tests are so far basic (load a canvas with a BMP and rectangle).

Versioning Info: Cosmos Devkit (latest version), C# 10.0 DotNet 6.0 (my system has DotNet 7.0 and 8.0 installed, but appears to just use 6.0), IL2CPU (Latest Version). Testing on VMWare Workstation 17.2. Devkit installed per instructions on [https://cosmosos.github.io/articles/Installation/DevKit.html](https://cosmosos.github.io/articles/Installation/DevKit.html)

**Comment by Gabolate** [comment]
I think those warnings shouldn't cause any issues, i usually get them and it dosen't affect the OS

**Comment by ThatGuyAstral** [comment]
those are just warnings, i get those too. they do not affect the OS (well atleast for me) and you can safely ignore them

**Comment by OtterConsultant** [comment]
Ok perfect. as long as they won't cause problems down the road should be good.

**Comment by zarlo** [comment]
yes, they are fine. but we should still clean up no longer needed plugs.
